src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,24 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  value: [],
+};
+
+export const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      const newBook = action.payload;
+      state.value.push(newBook);
+    },
```

Comment:
- [OPTIONAL] I highly recommend avoiding using the `push` method which mutates the array while adding a new book, instead you can use the Spread operator.

```suggestion
    addBook: (state, action) => {
      const newBook = action.payload;
      state.value = [...state.value, newBook];
    },
```